rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -591,29 +585,37 @@ void logPegoutTransactionCreated_invalidBtcTxHashOrOutpointValues_shouldFail(Sha
     /**********************************
      *  -------     UTILS     ------- *
      *********************************/
-    private static void assertEvent(List<LogInfo> logs, int index, CallTransaction.Function event, Object[] topics, Object[] params) {
-        final LogInfo log = logs.get(index);
-        assertEquals(LogInfo.byteArrayToList(event.encodeEventTopics(topics)), log.getTopics());
-        assertArrayEquals(event.encodeEventData(params), log.getData());
+    private void assertEvent(CallTransaction.Function event, Object[] topics, Object[] data) {
```

Comment:
same here, and `index` was always zero (also makes sense since we are unit testing, and we expect to only have one broadcasted log each time.)